lib/mongo/error/labelable.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+# encoding: utf-8
+
+# Copyright (C) 2019-2022 MongoDB Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module Mongo
+  class Error < StandardError
```

Comment:
The inheritance declaration should only happen in the file where the Error class is defined (lib/mongo/error.rb). This file should be included later such that the inheritance declaration isn't needed here.